### ISSUE TITLE
About How to Train 512 Model

### ISSUE THREAD

**Issue by Bai426** [SHI-Labs/FcF-Inpainting#21]
Hi, Author, I trained the 512 models (#17 ) but the model didn't converge until now. Since I only have 4 GPUs, I have to use a batch size of 16 or 8. I tried the lr from 0.0005 to 0.002, and different number of training image (--kimg 25000). But the training is unstable and I cannot get a converged model (i.e., FID score is from high to low and stable). Can you be kind to give some advice to handle this? Thank you very much!

Best,
Mike

**Comment by praeclarumjj3** [comment]
Hi @Bai426, thanks for your interest in our work. Training GANs is very much sensitive to the batch size used. As you said in an email, you could try [gradient accumulation](https://kozodoi.me/python/deep%20learning/pytorch/tutorial/2021/02/19/gradient-accumulation.html). It might work.

**Comment by Bai426** [comment]
Okay, thank you. Let me try what you suggested and see what will happen. 

Best,
Mike
